src/js/tabs/led_strip.js

```diff
@@ -100,7 +100,7 @@ led_strip.initialize = function (callback, scrollPosition) {
 
             $('.auxSelect').val(getModeColor(AuxMode, AuxDir));
 
-            $('.auxSelect').on('change', function() {
+            $('.auxSelect').change(function() {
```

Comment:
.change is deprecated.
.on('change', function() {}); is preferred.

Reply:
Done